### DISCUSSION TITLE
Why are there 3 ENA run accessions for each sample in Ag3?

### DISCUSSION THREAD

**Discussion by alimanfoo** [malariagen/vector-data#35] [Q&A]
As part of the Ag3 release there is a [catalog of ENA run accessions](https://malariagen.github.io/vector-data/ag3/download.html#raw-sequence-reads-fastq-format) which can be used to obtain the raw sequence reads. For most samples there are three ENA run accessions. Why is this?

**Comment by alimanfoo** [comment]
When we [sequence mosquitoes](https://malariagen.github.io/vector-data/ag3/methods.html#whole-genome-sequencing), we create a library for each individual mosquito, then the libraries are combined into a tagged multiplex, usually of libraries from 12 mosquitoes at the time Ag1000G was done. This multiplex is then run on three separate lanes of sequencing. The reason for the multiple lanes is to even out the variations in yield that we see between different sequencing runs. This means that, after the sequencing data are demultiplexed, each mosquito has data from three different sequencing runs. When these data are archived to ENA, they are accessioned as three separate runs.

(In fact it is not always three, some mosquitoes have only 1 or 2 runs, some have more than 3, depending on how the sequencing was done. But it's usually 3 runs per mosquito.)

When we run the [alignment and SNP calling pipeline](https://malariagen.github.io/vector-data/ag3/methods.html#alignment-and-snp-calling), we combine (merge) the data from all sequencing runs for each mosquito.

**Reply by clementfkent** [reply]
When I download a single vcf file - for example, AR0053-C.var.vcf.gz - where do I look to find out how m any individual mosquitoes data the file is based on? The reply above suggests it might be be 12?
